### PR TITLE
Externs cleanup for moment, D3

### PR DESCRIPTION
**Pull request by welchyd**
The moment externs had a couple functions that needed optional params
I disabled some linter errors and cleaned up some urls while I was in there

### CODE REVIEW

**Review by jsalankey — APPROVED** [review]

File: externs/d3-3.4.1.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
+/* eslint-disable jsdoc/require-returns-check */
```
Comment (wallw-bits) [inline externs/d3-3.4.1.js:1]: Why were these necessary? We do not lint externs.

Comment (welchyd) [reply externs/d3-3.4.1.js:1]: It's not really necessary, but some of the other externs have it
